_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -552,12 +554,29 @@ resource "aws_ebs_encryption_by_default" "ebs_encryption" {
 }
 ----
 
+[[configure_rds_encryption]]
+==== Configure RDS Encryption
+To comply with recommendation 2.3.1, if you use RDS it must be configured with link:http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.Encryption.html[RDS encryption] for all instances and in all of the enabled AWS regions within your AWS Account(s). You can set to `true` the Terraform `storage_encrypted` attribute for an RDS instance to implement the recommendation.
+
+For example:
+
+[source,hcl]
+----
+resource "aws_db_instance" "your_primary_db" {
+  storage_encrypted = true
+}
+----
+
+[[configure_s3_data]]
+==== Configure your S3 Data
+The CIS 1.4.0 Benchmark recommend a few additional steps to ensure your data is secured and accessible. The two recommendations relate both to S3 and prescribe that the MFA Delete is enabled on all S3 buckets in your accounts (2.1.3) and that all data in your S3 buckets is classified, discoverable and secured using Amazon Macie. You'll be able to find more details on how to enable these in link:/guides/upgrades/how-to-update-to-cis-14/[the migration guide to CIS 1.4.0].
```

Comment:
I think we can copy+paste the macie section here?

Reply:
I was thinking the same too. Good shout.

Reply:
Agreed, a bit of duplication here is the lesser evil, I think. That way we ensure we never have dead links.

Reply:
```suggestion
The CIS 1.4.0 Benchmark recommends a few additional steps to ensure your data is secured and accessible. There are two new recommendations that relate to S3:

* 2.1.3 prescribes that the MFA Delete is enabled on all S3 buckets in your accounts
* 2.1.4 recommends that all data in your S3 buckets be classified, discovered and secured
```